src/acm_common/truststore_utils.py

```diff
@@ -125,6 +125,35 @@ def generate_keystore(
     logger.info("Java KeyStore generated")
 
 
+def add_cert_and_key(
+    priv_key, cert_list, alias, priv_key_password=None
+):
+    """Add certificate and private key.
+
+    Args:
+        priv_key (str): The base64 PEM-encoded private key to store
+        cert_list (List of str): A list of base64 PEM-encoded certificates
+            signed by ACM PCA.
+            Multiple certificates should represent the chain in the right order.
+        alias (str): The alias under which to store the key pair
+        priv_key_password (str): The password to protect the private key with
```

Comment:
This parameter isn't used. It's rare (in my experience) to have private keys on disk protected by a passphrase, so I'd be happy to just see this removed from the function signature.

Reply:
Updated to remove password from function